cdap-docs/included-applications/source/etl/plugins/batchsources/file.rst

```diff
@@ -20,20 +20,9 @@ the logs in a TimePartitionedFileSet.
 
 .. rubric:: Properties
 
-**fileSystem:** Distributed file system to read in from.
-
-.. highlight:: xml
-
-**fileSystemProperties:** JSON of the properties needed for the
-distributed file system. The formatting needs to be as follows::
-
-  {
-    "<property name>": "<property value>"
-    ...
-  }
-
-For example, the property names needed for S3 are \"fs.s3n.awsSecretAccessKey\"
-and \"fs.s3n.awsAccessKeyId\".
+**fileSystemProperties:** A JSON string representing a map of properties needed for the
+distributed file system. For example, the property names needed for S3 are
```

Comment:
file.rst : It doesn't have the accessID and accessKey properties, correct? So, unlike the other two, these properties would not be automatically added to the fileSystemProperties?